Source/WebCore/platform/audio/cocoa/AudioDecoderCocoa.cpp

```diff
@@ -288,7 +289,7 @@ String InternalAudioDecoderCocoa::initialize(const String& codecName, const Audi
 
     m_inputFormatDescription = createAudioFormatDescription(*m_inputDescription, m_codecDescription.span());
 
-    m_outputDescription = CAAudioStreamDescription { double(config.sampleRate), uint32_t(config.numberOfChannels), AudioStreamDescription::Float32, CAAudioStreamDescription::IsInterleaved::No };
+    m_outputDescription = CAAudioStreamDescription { double(config.sampleRate), uint32_t(config.numberOfChannels), AudioStreamDescription::Float32, CAAudioStreamDescription::IsInterleaved::Yes };
```

Comment:
We should probably file a spec bug, can you do that?

Reply:
the spec lets the UA choose, there's no option to tell what you want out which is unfortunate.
In theory, how values are packed internally should be an implementation details and it should behave the same regardless.

However, `copyTo` has a very different behaviour if the origin is interleaved or not where a planeIndex of 0 now means "all planes"

Reply:
I've opened https://github.com/w3c/webcodecs/issues/859